src/util/entitlements/helpers.tsx

```diff
@@ -0,0 +1,10 @@
+export const hasEntitlement = (
+  required: string,
+  entitlements?: string[],
+  isFineGrained: boolean | null = false,
```

Comment:
```suggestion
  isFineGrained: boolean | null,
```
I think it is unexpected to apply a default here.